src/store/WalletStore.ts

```diff
@@ -86,7 +86,7 @@ class WalletStore {
         )
 
         autorun(() => {
-            if (this.seed && !this.locked) {
+            if (this.seed && this.seed?.length > 0 && !this.locked) {
```

Comment:
We can remove this.seed condition =>this.seed?.length > 0 && !this.locked

Reply:
I try to use >this.seed?.length > 0 , but I will appear typescript error
can you help me solve it?